SCClassLibrary/Common/Streams/Patterns.sc

```diff
@@ -343,6 +343,8 @@ Pbind : Pattern {
 	var <>patternpairs;
 	*new { arg ... pairs;
 		if (pairs.size.odd, { Error("Pbind should have even number of args.\n").throw; });
+        if (pairs.asEvent[\instrument].rank != 0, 
```

Comment:
This creates unnecessary objects for the garbage collector to deal with, just for a test. It also won't work for syntactic cases like: `Pbind([\instrument\freq], [[\be, \bo], [400, 600]]`.